Riot/Modules/DeviceVerification/Verify/Scanning/QRCode/Reader/QRCodeReaderViewController.swift

```diff
@@ -0,0 +1,270 @@
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import ZXingObjC
+
+protocol QRCodeReaderViewControllerDelegate: class {
+    func qrCodeReaderViewController(_ viewController: QRCodeReaderViewController, didFound payloadData: Data)
+    func qrCodeReaderViewControllerDidCancel(_ viewController: QRCodeReaderViewController)
+}
+
+/// QRCodeReaderViewController is a view controller used to scan a QR code
+/// Some methods are based on [ZXing sample](https://github.com/zxingify/zxingify-objc/blob/master/examples/BarcodeScannerSwift/BarcodeScannerSwift/ViewController.swift)
+final class QRCodeReaderViewController: UIViewController {
```

Comment:
This QRCodeReader could be its own module instead of being under `Modules/DeviceVerification/Verify/Scanning/QRCode/Reader/`